cumulusci/robotframework/pageobjects/ObjectManagerPageObject.py

```diff
@@ -0,0 +1,230 @@
+from cumulusci.robotframework.utils import capture_screenshot_on_error
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+from selenium.webdriver.common.keys import Keys
+
+object_manager = {
+    "button": "//input[@title='{}']",
+    "input": "//input[@id='{}']",
+    "select_related": "//select[@id = '{}']",
+    "select_related_option": "//select[@id = 'DomainEnumOrId']/option[@value='{}']",
+    "search_result": "//tbody/tr/td/a/span[contains(text(),'{}')]",
+    "formula_txtarea": "//textarea[@id = '{}']",
+    "object_result": "//th/a[text()='{}']",
+    "link-text": "//a[contains(text(),'{}')]",
+    "button-with-text": "//button[contains(text(),'{}')]",
+    "frame_new": "//iframe[contains(@name, '{}') or contains(@title, '{}')]",
+    "action_menu": "//a[contains(@class,'rowActionsPlaceHolder')]",
+    "action_menu_item": "//span[text() = '{}']",
+    "delete_confirm_btn": "//button[contains(@class,'forceActionButton')]",
+}
+# All common elements
+search_button = object_manager["input"].format("globalQuickfind")
+currency_locator = object_manager["input"].format("dtypeC")
+next_button = object_manager["button"].format("Next")
+save_button = object_manager["button"].format("Save")
+text_locator = object_manager["input"].format("dtypeS")
+formula_locator = object_manager["input"].format("dtypeZ")
+checkbox_option = object_manager["input"].format("fdtypeB")
+formula_txtarea = object_manager["formula_txtarea"].format("CalculatedFormula")
+check_syntax = object_manager["button"].format("Check Syntax")
+actions_menu = object_manager["action_menu"]
+action_item_delete = object_manager["action_menu_item"].format("Delete")
+confirm_delete = object_manager["delete_confirm_btn"]
+lookup_locator = object_manager["input"].format("dtypeY")
+
+
+@pageobject(page_type="ObjectManager")
+class ObjectManagerPage(BasePage):
+    """A page object representing the Object Manager of an object.
+        Example
+        | Go to page   ObjectManager  Contact
+        """
+
+    def _go_to_page(self):
+        url_template = "{root}/lightning/setup/ObjectManager/home"
+        url = url_template.format(
+            root=self.cumulusci.org.lightning_base_url, object_name=self.object_name
```

Comment:
No need to include object_name here, since it's not part of url_template

Reply:
Agree, done

Reply:
@rjanjanam It's still here -- do you need to push a commit?